src/fontra/views/editor/scene-controller.js

```diff
@@ -1148,6 +1168,49 @@ export class SceneController {
     });
   }
 
+  async closeContour() {
+    const openContours = this.contextMenuState.openContourSelection;
+    const curveType = this.experimentalFeatures.quadPenTool ? "quad" : "cubic";
+    await this.editLayersAndRecordChanges((layerGlyphs) => {
+      for (const layerGlyph of Object.values(layerGlyphs)) {
+        const path = layerGlyph.path;
+        for (const contourIndex of openContours) {
+          // close open contour
+          path.contourInfo[contourIndex].isClosed = true;
+          if (curveType === "cubic") {
+            this.closeContourEnsureCubicOffCurves(path, contourIndex);
+          }
+        }
+      }
+      return "Close Contour" + (openContours.length > 1 ? "s" : "");
```

Comment:
Not sure if we do this elsewhere, but why not use the same translation as for the menu item.

Reply:
Thanks. Done.